src/components/app/Root.jsx

```diff
@@ -56,6 +37,10 @@ const Root = () => {
     );
   }
 
+  if (!isAppDataHydrated) {
```

Comment:
[inform] Accounts for both user profile hydration and whether there's a notices redirect.